docs/glossary.md

```diff
@@ -189,6 +189,14 @@ Tickets that reached the end of their window without being called to vote - thes
 
 A mechanism for obtaining free (testnet) coins.
 
+#### Fork
```

Comment:
As this section is describing **software** forks and not **chain** forks, using Bitcoin/Bitcoin Cash as an example is rather confusing because that represents both a software and a chain fork. For ultimate clarity, I would suggest using an example completely outside of cryptocurrency - eg. Ubuntu as a fork Debian, LibreOffice as a fork of OpenOffice, Plex as a fork of XBMC, or if you want to go really retro and shout-out to some of the c0 guys, OpenBSD as a fork of NetBSD.

Reply:
The section actually blends the two definitions, making it quite hard to distinguish which is which. 

Perhaps we could simplify by splitting these very clearly into two distinct sections:

> The term fork has two common meanings:
>  - Source code forks are...
>  - Chain forks are ....

Reply:
I see what you mean. Took a stab at rewording the first paragraph:

>In the context of blockchains, the term fork has two common meanings; it can refer to either a code fork or a chain fork. A [code fork](https://en.wikipedia.org/wiki/Fork_(software_development)) is when developers take a copy (fork) of the source code of a project and begin independently developing it, creating a separate version of the software. For instance, the OpenBSD operating system is a fork of NetBSD. If the state of the blockchain is also forked, as was the case with Bitcoin Cash, it is also considered a “chain fork”. In this case, owners of coins on the original chain will have equivalent coins on the new chain when the chains diverge. Decred began its life as a source code fork of [btcd](https://github.com/btcsuite/btcd), an alternate full node implementation of Bitcoin. Significant changes were then made before launch, including modifications to the consensus rules, a Proof-of-Stake (PoS) layer to enable on-chain governance, a new hashing algorithm and other innovations. It is arguable whether Decred is a source code fork, considering the scope of modifications made. Decred is not a chain fork, as it did not copy the state of the Bitcoin blockchain, and instead started from its own genesis block.